src/pages/FinalCheckout.tsx

```diff
@@ -147,295 +161,205 @@ export default function FinalCheckout() {
-              Payment Information
-            </Typography>
-            <Grid container spacing={2}>
-              <Grid item xs={12}>
-                <TextField 
-                fullWidth 
-                label="Card Number" 
-                required 
-                variant="outlined"
-                value={formData.cardNumber}
-                onChange={handleInputChange('cardNumber')}
-                error={tabValue === 1 && formData.cardNumber === ''}
-                helperText={tabValue === 1 && formData.cardNumber === '' ? 'This field is required' : ''}
-                 />
-              </Grid>
-              <Grid item xs={6}>
-                <TextField 
-                fullWidth 
-                label="Expiration Date"
-                required 
-                variant="outlined"
-                value = {formData.expirationDate} 
-                onChange={handleInputChange('expirationDate')}
-                error={tabValue === 1 && formData.expirationDate === ''}
-                helperText= {tabValue === 1 && formData.expirationDate === '' ? 'This field is required' : ''}                 
-                />
-              </Grid>
-              <Grid item xs={6}>
-                <TextField 
-                fullWidth 
-                label="CVV" 
-                required 
-                variant="outlined"
-                value = {formData.cvv} 
-                onChange={handleInputChange('cvv')}
-                error={tabValue === 1 && formData.cvv === ''}
-                helperText= {tabValue === 1 && formData.cvv === '' ? 'This field is required' : ''}                 
-                />
-              </Grid>
-            </Grid>
-          </Paper>
-        </TabPanel>
-        </Box>
+      <Typography variant="h6" sx={{ mb: 1, textAlign: 'left', fontWeight: 500 }}>
+        Required Details
+      </Typography>
+
+      <Grid item xs={12} sx={{ mb: 3 }}>
+        <TextField
+          fullWidth
+          label="Workflow Name"
+          required
+          variant="outlined"
+          value={formData.workflowName}
+          onChange={handleInputChange('workflowName')}
+          error={tabValue === 1 && formData.workflowName === ''}
+          helperText={tabValue === 1 && formData.workflowName === '' ? 'This field is required' : ''}
+        />
+      </Grid>
+
+      <Typography variant="h6" sx={{ mb: 1, textAlign: 'left', fontWeight: 500 }}>
+        Contact Information
+      </Typography>
+      
+      <Typography variant="body2" sx={{ mb: 2, color: 'text.secondary', textAlign: 'left' }}>
+        Please verify your contact details below. These are read-only.
+      </Typography>
+
+      {/* Non-editable contact info */}
+      <Grid container spacing={0.5} direction="column">
+        <Grid item xs={12} sx={{ mb: 1 }}>
+          <TextField
+            label="Username" // Will need to refactor to display Family name + Given name from user data

```

Comment:
Sounds good to me! Seems like keycloak just uses the email address as the username anyway. Except maybe don't call the field username; just call it Your Name or something.